readme.md

```diff
@@ -258,6 +258,12 @@ As in: OOPs I used Java!
 
 ---
 
+**Q:** Where do slow queries go to retire?
+
+**A:** A little cabin by the latency.
```

Comment:
I don't get it? Is there a Laten Sea or something?

Reply:
not that I know of.

Reply:
I re-worked it a bit, see if that makes more sense.